src/redux/actions/dictionaries/dictionaryActions.js

```diff
@@ -1,4 +1,4 @@
-import { ADDING_DICTIONARY, FETCHING_ORGANIZATIONS, ADDING_DICTIONARY_FAILURE, FETCHING_DICTIONARIES, IS_FETCHING } from './../types';
+import { ADDING_DICTIONARY, FETCHING_ORGANIZATIONS, ADDING_DICTIONARY_FAILURE, FETCHING_DICTIONARIES, IS_FETCHING, FETCHING_DICTIONARY } from './../types';
```

Comment:
in a case like this where the types are too many you can always do 
`import * as types from '../../yourTypesLocation` would be much neater.
 when calling the types you can then do types. ADDING_DICTIONARY

Reply:
A nice suggestion, but I think that would be done as a sub ticket because it will require me to make so many changes to that whole file.
What do you think

Reply:
true

Reply:
true i agree